modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/undo/UndoHistory.js

```diff
@@ -14,7 +14,6 @@
 
 import {ClayButtonWithIcon} from '@clayui/button';
 import ClayDropDown, {Align} from '@clayui/drop-down';
-import ClayDropDownDivider from '@clayui/drop-down/lib/Divider';
```

Comment:
Hmm, we could investigate creating an eslint rule for this so that people don't reach into these packages like this. At least for clay.

Reply:
Yes. Apparently some people is copy pasting that and propagating the error :-(

Reply:
I already got this error by using autocomplete from VSCode. For example, if you autocomplete <ClayButton, it will automatically require ClayButton but taking from the lib folder instead of considering the module's entry point. I don't know if this thing can be avoidable or not

Reply:
> it will automatically require ClayButton but taking from the lib folder

Oh interesting, perhaps we can look into this and maybe it's how we have configured TS in clay. I will spawn an issue off for this in Clay to investigate.

Reply:
I'm not really sure if we can solve this, I think it's because of the generated path of the types, for example `Divider.tsx` will generate `Divider.d.ts`, so the Divider reference instead of going to ` index.d.ts` it will go to the final file because our index is just an export.

I'm not sure if this would be more of a VSCode side configuration but I don't know how we can change it in Clay, since we don't bundle the clay in just one file and I don't think there's any way to change the result of the types, we can search if there is a way but I find it difficult.